package.json

```diff
@@ -49,12 +49,15 @@
   "version": "10.0.0-rc.0",
   "dependencies": {
     "@angular/animations": "^10.0.0-rc.2",
+    "@angular/benchpress": "^0.2.0",
     "@angular/common": "^10.0.0-rc.2",
     "@angular/compiler": "^10.0.0-rc.2",
     "@angular/core": "^10.0.0-rc.2",
+    "@angular-devkit/build-optimizer": "^0.900.6",
```

Comment:
Can we use `^0.1000.2` as that matches with other CLI packages we have installed?

Reply:
Optional Nit: Usually we'd put these (becnhpress, BO, terser) dependencies that aren't used in components into `devDependencies`, but that shouldn't be an issue here.

Reply:
@devversion Did you mean `^0.1000.0-rc.2` for the build-optimizer? Also, I'm going to move those three deps to devDependencies like you mentioned :+1:

Reply:
@wagnermaciel yeah! thanks I just typed the version manually but got it wrong